diff --git a/services_views.resource.inc b/services_views.resource.inc
index dc07d09..d36732f 100644
--- a/services_views.resource.inc
+++ b/services_views.resource.inc
@@ -178,7 +178,7 @@ function services_views_execute_view($view_info, $view = NULL, $display_id = NUL
         // Create helper variables.
         $output[$index]->$target_key = array();
         if ($field->field_info['cardinality'] == 1) {
-          $output[$index]->$target_key = '';
+          $output[$index]->{$target_key}[] = '';
         }
         $obj = &$output[$index]->$target_key;
         $format = ($field->options['type'] == 'services') ? 'raw' : 'rendered';
